apps/studio/src/server/modules/folder/folder.router.ts

```diff
@@ -76,7 +76,13 @@ export const folderRouter = router({
         .updateTable("Resource")
         .where("Resource.id", "=", resourceId)
         .where("Resource.siteId", "=", Number(siteId))
-        .where("Resource.type", "=", "Folder")
+        .where((eb) =>
+          eb("Resource.type", "=", "Folder").or(
+            "Resource.type",
+            "=",
+            "Collection",
+          ),
+        )
```

Comment:
nit: can be simplified with


```suggestion
        .where("Resource.type", "in", ["Folder", "Collection"])
```